### DISCUSSION TITLE
I added a plugin, but it was not included in the application after the MVN package step

### DISCUSSION THREAD

**Discussion by LuckyPickleZZ** [dbeaver/dbeaver#13200] [DBeaver Development]
Hello, I have added a plugin in the project. The pom.xml was also altered. But after MVN package, it was not included in the application. The path of the application is /dbeaver/product/community/target/products/org.jkiss.dbeaver.core.product/macosx/cocoa/x86_64/Dbeaver.
<img width="514" alt="image" src="https://user-images.githubusercontent.com/36879124/125726709-70d183e2-45ba-45dc-a200-499c671653c4.png">
pom.xml of my plugin

<img width="447" alt="image" src="https://user-images.githubusercontent.com/36879124/125726771-56e6120a-da49-4b8a-a03a-f1fe44c8162b.png">
plugins/pom.xml

Is there any configuration I have missed?

**Comment by LonwoLonwo** [comment]
Hello @LuckyPickleZZ 
What about your plugin manifest file? What does it contain?
Adding plugin into features/org.jkiss.dbeaver.db.feature/feature.xml is also important.

**Reply by LuckyPickleZZ** [reply]
<img width="588" alt="image" src="https://user-images.githubusercontent.com/36879124/125900083-d539d206-9df8-4ad0-bc7e-d23e296fe059.png">
My manifest.mf is here.
And I really appreciate your advice! After I added it to feature.xml, it did work !!

Also, could you please check this discussion ? If you can give some answers or suggestions, I will be very grateful.

https://github.com/dbeaver/dbeaver/discussions/13175
